modules/customizing-the-web-console-limits.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies:
+//
+// * web_console/customizing-the-web-console.adoc
```

Comment:
module seems unnecessary when the line looks like it's included in web_console/customizing-the-web-console.adoc below?

Reply:
I feel like this could probably just be added as a NOTE rather than a module or having its own heading?

Reply:
Thanks, @abrennan89. Yes, I thought I had removed references to the module and deleted it. is it still existing?

Reply:
Yep, still showing up as a new file. I'd try deleting it again locally and recommitting and pushing your changes.

Reply:
deleted file.